pom.xml

```diff
@@ -183,7 +183,7 @@
                 <plugin>
                     <groupId>org.apache.felix</groupId>
                     <artifactId>maven-bundle-plugin</artifactId>
-                    <version>2.3.5</version>
+                    <version>3.5.0</version>
```

Comment:
Any reason for changing version?

Reply:
Fixed with commit : 7cf8e072f7b31ab450a45af082816aeda2f266cc (changed due to an issue. It is fixed now)